CHANGELOG.md

```diff
@@ -96,6 +97,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ##### Ferenia
 
+- Added: The missile tank in the Pitfall Puzzle Room can now be obtained with Flash Shift.
```

Comment:
This should get moved to the 6.4.0 section.

Reply:
This shouldnt exist here at all, this is part of the other PR: #5380

Reply:
I believe I've remedied the problem. I'll have to go and check the other PR as well.